romanian/vara.txt

```diff
@@ -0,0 +1 @@
+acum Ginel merge la plaja sa vada fete goale
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.